ui/app/components/ui/boxed-message/boxed-message.component.js

```diff
@@ -0,0 +1,55 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+export default class BoxedMessage extends PureComponent {
+  static propTypes = {
+    boxClass: PropTypes.string,
+    iconSource: PropTypes.string,
+    iconSize: PropTypes.string,
+    height: PropTypes.string,
+    width: PropTypes.string,
+    message: PropTypes.string,
+  }
+
+  static defaultProps = {
+    height: 'auto',
+    width: '100%',
+    iconSize: '14px',
+    iconSource: null,
+    boxClass: '',
+    message: '',
+  }
+
+  render () {
+    const {
+      iconSize,
+      iconSource,
+      height,
+      width,
+      boxClass,
+      message,
+    } = this.props
+
+    return (
+      <div
+        className={`boxed-message__container ${boxClass}`}
+        style={{
+          height,
```

Comment:
Why the inline style? Styling this via SCSS would be more flexible, and would let us handle different screen sizes and such.

Reply:
How do we allow passing of height and width as props while using SCSS for height and width?

Reply:
We wouldn't; we'd set it in the styles. That would let us set _different_ width/height for different viewport sizes using media queries. This width looks pretty silly on fullscreen.

Reply:
I do think this is worth fixing in this PR, as the vast majority of our UI can be viewed in fullscreen, and that means we need the messages to scale based on the viewport. That's not feasible if height and width is specified via props (it's possible to do this via React of course, but it's awkward and inefficient)

Reply:
I'm a little confused by these comments.

You ask "Why the inline style? Styling this via SCSS would be more flexible" Then in response to "How do we allow passing of height and width as props while using SCSS" you say "We wouldn't; we'd set it in the styles."

I must be misinterpretting or missing something, because it feels like the first comment implied "don't use styles" and the second implies "use styles". I'm sure you are making sense and I am just confused... can you further clarify?

Meanwhile, I changed the default width in 121f69b4a, to fix the width problem you describe.

Reply:
oh, wait, I think I know what you mean about the styles... new commit incoming

Reply:
Does 1ae753d05 come closer to what you were suggesting?

Reply:
Not really 😅 What I meant is that it shouldn't be set as a prop at all - it shouldn't be in React. It's better for it to be set in the SCSS styles, because they know the viewport. We don't know the viewport in this context.

Reply:
The `style` prop remains, though it doesn't appear to be used.

Perhaps we can just remove it for now? And add a `className` prop if further customization is needed.